File: password/template/password.cpp
#include <bits/stdc++.h>
using namespace std;

string decipher(vector<string> &strings) {

}

int main() {
  int n;
  cin >> n;

  vector<string> strings(n);
  for (int i = 0; i < n; i++)
    cin >> strings[i];

  cout << decipher(strings) << endl;
  return 0;
}

File: password/template/password.rs
use std::collections::HashSet;
use std::io::{self, BufRead};

fn decipher(strings: &Vec<String>) -> String {

}

fn main() {
    let stdin = io::stdin();
    let mut stdin_iterator = stdin.lock().lines();

    let n = stdin_iterator
        .next()
        .unwrap()
        .unwrap()
        .trim()
        .parse::<usize>()
        .unwrap();

    let mut strings: Vec<String> = Vec::with_capacity(n);
    for _ in 0..n {
        let string = stdin_iterator.next().unwrap().unwrap().trim().to_string();
        strings.push(string);
    }

    println!("{}", decipher(&strings));
}
